packages/app/src/components/DataTable.tsx

```diff
@@ -1,90 +1,256 @@
+        }
+    ]
+
+    if (data === null) {
+        return null;
+    }
+
     return (
-        <Dialog
-            open={true}
-            onClose={props.close}
-            fullWidth
-            sx={{height: "100%", width: "100%"}}
-        >
-            <DataGrid
-                columns={columns}
-                rows={tableRows}
-                autoHeight
-                disableSelectionOnClick
-                initialState={{
-                    pagination: {
-                        page: 0,
-                        pageSize: 10,
-                    },
-                  }}
-                paginationMode="client"
-                rowsPerPageOptions={[5, 10, 20]}
-            ></DataGrid>
-            <div style={{display: "flex", justifyContent: "flex-end", margin: "10px"}}>
-                <Button color="inherit" size="medium" variant="outlined" onClick={props.close}>Close</Button>
-            </div>
-        </Dialog>
+        <>
+            <Box sx={{display: "flex", justifyContent: "space-between"}}>
+                <Box sx={{width: "50%", margin: "20px"}}>
+                    <div style={headerCSS}>
+                        <h2>UpSet Data Table</h2>
+                        <DownloadButton onClick={() => downloadElementsAsCSV(tableRows, ["elementName", "size"], "upset2_datatable")} />
+                    </div>
+                    <DataGrid
+                        columns={dataColumns}
+                        rows={tableRows}
+                        autoHeight
+                        disableSelectionOnClick
+                        initialState={{
+                            pagination: {
+                                page: 0,
+                                pageSize: 10,
+                            },
+                        }}
+                        paginationMode="client"
+                        rowsPerPageOptions={[5, 10, 20]}
+                    ></DataGrid>
+                </Box>
+                <Box sx={{width: "30%", margin: "20px"}}>
+                    <div style={headerCSS}>
+                        <h2>Visible Sets</h2>
+                        <DownloadButton onClick={() => downloadElementsAsCSV(visibleSetRows, ["setName", "size"], "upset2_visiblesets_table")} />
+                    </div>
+                    <DataGrid
+                        columns={setColumns}
+                        rows={visibleSetRows}
+                        autoHeight
+                        disableSelectionOnClick
+                        initialState={{
+                            pagination: {
+                                page: 0,
+                                pageSize: 10,
+                            },
+                        }}
+                        paginationMode="client"
+                        rowsPerPageOptions={[5, 10, 20]}
+                    ></DataGrid>
+                </Box>
+                <Box sx={{width: "30%", margin: "20px"}}>
```

Comment:
The widths on these boxes adds to over 100% does that cause horizontal scrolling on smaller screens? Perhaps we should just make it equal 100

Reply:
I think since they are in a flex box there isn't any horizontal scrolling. I updated it to be 50 25 25, however.